ansible/roles/postgresql/tasks/main.yml

```diff
@@ -92,17 +92,52 @@
   when:
     pg_install == True
 
+- name: Create PostgreSQL ICDS users
```

Comment:
also if the ICDS users are in the `posetgres_users` list then they'll get added to pgbouncer userlist